extra/onnx.py

```diff
@@ -13,16 +14,16 @@
   from onnx.mapping import TENSOR_TYPE_TO_NP_TYPE
   tensor_dtype_to_np_dtype = lambda x: TENSOR_TYPE_TO_NP_TYPE[x]
 
-# global numpy cache for parameters
-numpy_cache = {}
-def safe_numpy(t) -> np.ndarray:
+# global Tensor -> list cache for parameters
+to_python_cache = {}
```

Comment:
can it work with `@functools.lru_cache` and don't need to manually maintain cache?